bonfire/graylog_api.py

```diff
@@ -123,6 +122,7 @@ def get(self, url, **kwargs):
 
     def search(self, query, fetch_all=False):
         sort = None
+        self.host_tz = self.host_timezone()
```

Comment:
is there a reason why we determine the timezone every time we search (we might call the search method several times, after all, when in follow mode) instead of say placing that call in the init function, in case of no passed `host_tz` argument (effectively `host_tz == None`)?

Reply:
if we move the call to the init function, we could in particular add a test that

- registers the necessary API mocks, with some timezone
- intializes a graylog API object without setting the timezone
- checks that the timezone is fetched and set correctly on the API object

do you think such a test sounds reasonable?